bower.json

```diff
@@ -11,6 +11,7 @@
   ],
   "description": "Origami module for FT tracking.",
   "dependencies": {
-    "ftdomdelegate": "^4.0.0"
+    "ftdomdelegate": "^4.0.0",
+    "is-circular": "tjmehta/is-circular#^1.0.2"
```

Comment:
`tjmehta/is-circular` is quite a small package without many contributors. For security maybe we should implement directly in `o-tracking` or lock down to a specific version?

Reply:
Sure thing, I coded one up myself but decided to use their version instead for some reason --
https://gist.github.com/JakeChampion/e1c593b67c59d0f75c7ca3c1cb86a21f